Tabloid/Controllers/TagController.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Tabloid.Repositories;
+using Tabloid.Models;
+using Microsoft.AspNetCore.Authorization;
+
+namespace Tabloid.Controllers
```

Comment:
Users must be authorized to view data, make sure to add the [Authorize] attribute to this class or to each individual methods